config/core/test-configs-chromeos.yaml

```diff
@@ -91,8 +91,8 @@ device_types:
     mach: x86
     arch: x86_64
     boot_method: depthcharge
-    filters: &x86-chromebook-filters
-      - passlist: {defconfig: ['x86-chromebook']}
+    filters: &x86-pineview-chromebook-filters
+      - passlist: {defconfig: ['cros://chromeos-5.10/x86_64/chromeos-intel-pineview.flavour.config']}
```

Comment:
This will only run 5.10 builds. I think we should just use `chromeos-intel-pineview` in the filter instead.

The `&x86-pineview-chromebook-filters` anchor can probably be dropped as each Chromebook type will be using a particular filter.  This was intended to be used with a generic fragment for all x86 Chromebooks.

Reply:
Updated filter
Maybe we can leave anchor? Because hatch uses pineview too (according to SDK)

Reply:
Yeah, or that can be added when `hatch` is added. But `asus-C436FA-Flip-hatch_chromeos` will be first in the file anyway due to alphabetical order. So it's probably better just to drop the anchor to avoid unnecessary noise in the next commits (they can also cause issues when rebasing / merging in `main`).

Reply:
Ok, updated